src/bicep/modules/SentinelSolution.bicep

```diff
@@ -0,0 +1,20 @@
+param workspaceName  string
+param workspaceId string
```

Comment:
Since we're gonna pull in the Log Analytics Workspace resource ID from the existing resource declaration we won't need this anymore.